eng/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: Repository
+    type: string
+    default: $(Build.Repository.Name)
+  - name: Prefix
+    type: string
+  - name: CIConventionOptions
+    type: string
+    default: ''
+  - name: UPConventionOptions
+    type: string
+    default: ''
+  - name: TestsConventionOptions
+    type: string
+    default: ''
+
+steps:
+  - template: install-pipeline-generation.yml
+  - script: |
```

Comment:
Can you please split this into multiple lines? Maybe our usual one argument per line approach.

Reply:
Done.